### ISSUE TITLE
Option to Export Posts as PDFs

### ISSUE THREAD

**Issue by carinapilar** [Automattic/wp-calypso#56325]
### What

Add an option to export content in a readable way, such as PDF files.

### Why

Right now users only have the option to create an XML when exporting their site's content, but sometimes they want a readable way, we get requests for PDFs and books for example.

Requested on 31035514-hc

**Comment by Aurorum** [comment]
Hi @carinapilar, just out of curiosity, why can't the user use the existing [Print option](https://jetpack.com/support/sharing/) which can be added to their blog for this? Are they looking to export multiple posts as PDFs instead? 

<img width="1291" alt="Screenshot 2021-09-18 at 11 34 03" src="https://user-images.githubusercontent.com/43215253/133885898-586cb6a8-6d04-4375-a176-a76de7a2cceb.png">

**Comment by carinapilar** [comment]
> Are they looking to export multiple posts as PDFs instead?

Yes, it would be more like to print all posts at once. I would also guess we wouldn't give the user the same result they get from the browser's printing option (which it feels to me it doesn't look great most of the time), we could print just the post's content with its original format.

**Comment by donalirl** [comment]
Just to mention there are services like https://www.blogbooker.com/ that can import an XML and output the posts as a PDF.
